DESCRIPTION

```diff
@@ -47,7 +47,8 @@ Depends:
 Imports:
     utils
 Suggests:
-    testthat
+    testthat (>= 3.0.0)
```

Comment:
Please add yourself as an author in the `DESCRIPTION` file.

Reply:
Done, thank you